meta-mel/conf/distro/include/jobs-heuristic.inc

```diff
@@ -0,0 +1,95 @@
+# Set bitbake threads and make jobs based upon available memory per CPU core.
+# Based on https://elinux.org/images/d/d4/Goulart.pdf, a talk done by Mario Domenech
```

Comment:
So we've implemented the algo as per page 50 of the mentioned study? If yes, that looks a bit different to me. Comments to follow on the exact scenarios.